docs/topics/installing-vs-code-extension.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+
+[id="installing-vs-code-extension_{context}"]
+= Installing the {ProductShortName} extension on Visual Studio Code
+
+You can install the {ProductShortName} extension on Visual Studio Code (VS Code) on Linux, Windows, or macOS.
```

Comment:
"on" > "for"

You might want to remove "on Linux, Windows, or macOS" because the user can assume that any platform that supports VS Code will support the extension.

Reply:
Since removing the phrase left me with "You can install the {ProductShortName} extension on Visual Studio Code (VS Code)." -- a rather uninformative sentence, I decided to go with "...on any platform that supports VS Code." WDYT?